inc/Integrations/Google/Sheets/GoogleSheetsIntegration.php

```diff
@@ -18,6 +18,11 @@ public static function register_blocks(): void {
 
 		foreach ( $data_source_configs as $config ) {
 			$data_source = GoogleSheetsDataSource::from_array( $config );
+	
+			if ( false === $config['service_config']['enable_blocks'] ) {
+				continue;
+			}
+			
 			self::register_blocks_for_google_sheets_data_source( $data_source );
 			self::register_loop_blocks_for_google_sheets_data_source( $data_source );
```

Comment:
Same thing here. Take it or leave it
```suggestion
	
			if ( false !== $config['service_config']['enable_blocks'] ) {
				self::register_blocks_for_google_sheets_data_source( $data_source );
				self::register_loop_blocks_for_google_sheets_data_source( $data_source );
			}
```